src/pages/library/Code.js

```diff
@@ -85,34 +89,42 @@ export const MethodRepoTile = () => {
   ])
 }
 
+const getFeaturedMethods = async signal => {
+  const [featuredMethods, methodsRepoMethodsList, dockstoreMethodsList] = await Promise.all([
+    Ajax(signal).FirecloudBucket.getFeaturedMethods(),
+    Ajax(signal).Methods.list({ namespace: 'gatk' }),
+    Ajax(signal).Dockstore.listTools({ organization: 'gatk-workflows' })
+  ])
+
+  const methodsRepoMethodDetails = _.flow(
+    _.groupBy(method => `${method.namespace}/${method.name}`),
```

Comment:
Nit: For consistency with below `_.map`:
```suggestion
    _.groupBy(({ namespace, name }) => `${namespace}/${name}`),
```

Reply:
Done in [71356c2](https://github.com/DataBiosphere/terra-ui/pull/3127/commits/71356c27c257695e216f095763cb789b6ce51e62).